certification/internal/engine/engine.go

```diff
@@ -96,12 +99,12 @@ func (c *CraneEngine) ExecuteChecks() error {
 		return fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
 	}
 
-	// only write these files to disk for container checks
-	if !c.IsBundle {
-		if err := writeCertImage(img); err != nil {
-			return err
-		}
+	// write json files to disk to be used to send to pyxis after all checks are complete
```

Comment:
this comment is probably no longer valid, since we are going to write this file for bundles and non `container --submit` cases as well.